weaviate/collections/batch/grpc.py

```diff
@@ -233,6 +234,8 @@ def __translate_properties_from_python_to_grpc(
                         prop_name=key, values=values, values_bytes=values_bytes
                     )
                 )
+            elif isinstance(val, GeoCoordinate):
+                non_ref_properties.update({key: val._to_dict()})
```

Comment:
`batch.proto` wasn't updated to use the new `Properties` struct so this won't work as intended. I left batch to depend on `google.protobuf.Struct` to minimise the number of changes required in the previous PR, we need to make a decision on how to refactor `batch.proto` to accept `GeoCoordinate` types but I don't think it needs to be in this PR (nor in its sibling PR within weaviate-core)

Reply:
> batch.proto wasn't updated to use the new Properties struct so this won't work as intended


Did you notice the tests that show that it works? 😅

Reply:
Oh damn! I didn't see the `insert_many` in there! I didn't see any server-side changes for it so assumed you weren't using it! Sorry for being so confidently incorrect 😂

Reply:
no worries. I DID add server side changes for batch at first and only noticed afterwards that it expects a dictionary and then I can just send it without any changes directly from the client